compiler/modules/front.syntax/astString.bal

```diff
@@ -648,8 +648,7 @@ function wordsToLines(Word[] s) returns string[] {
             firstInLine = true;
             lines.push("".'join(...buf));
             buf.setLength(0);
-            // JBUG #33532 should use `_` here
-            foreach int i in 0 ..< level {
```

Comment:
Why wasn't this giving a unused var warning? Though should be a JBUG itself.

Reply:
It is a special case in this case, Please refer to this: https://github.com/ballerina-platform/ballerina-spec/issues/356#issuecomment-964781409